### PR TITLE
Migrate to net framework

### PR DESCRIPTION
**Pull request by marabooy**

### CODE REVIEW

File: src/Microsoft.AspNetCore.OData.Nightly.nuspec [hunk 1]
```diff
@@ -4,8 +4,8 @@
     <id>Microsoft.AspNetCore.OData</id>
     <title>Microsoft ASP.NET Core 3.x and 5.x for OData v4.0</title>
     <version>$VersionFullSemantic$-Nightly$NightlyBuildVersion$</version>
-    <authors>Microsoft</authors>
-    <copyright>&#169; Microsoft Corporation. All rights reserved.</copyright>
```
Comment (xuzhg) [inline src/Microsoft.AspNetCore.OData.Nightly.nuspec:8]: Does this change block the release for aspnetcore.odata?

Comment (xuzhg) [reply src/Microsoft.AspNetCore.OData.Nightly.nuspec:8]: As we discussed, we only port newtonsoftjson project to .net foundation? right?

Comment (marabooy) [reply src/Microsoft.AspNetCore.OData.Nightly.nuspec:8]: Hi Sam, currently this should not be a blocker as the old pipeline will be running to sign the current binary.
Once we move the 7x series it should proceed normally. I will revert this to the original until we move the 7x.

File: azure_pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  - task: NuGetToolInstaller@1
+    displayName: Use NuGet >=5.2.0
+    inputs:
+      versionSpec: '>=5.2.0'
+      checkLatest: true
+  - task: UseDotNet@2
+    displayName: Use .NET Core sdk 3.1
+    inputs:
+      version: 3.x
+  - task: UseDotNet@2
```
Comment (xuzhg) [inline azure_pipelines.yml:29]: Can we move the yml files to a subfolder?

**Review by xuzhg — APPROVED** [review]

**Pull request merged**